app/src/main/java/net/skyscanner/backpack/demo/compose/BottomSheetStory.kt

```diff
@@ -65,3 +69,39 @@ fun BottomSheetStory(
         },
     )
 }
+
+@Composable
+@BottomSheetComponent
+@ComposeStory(name = "Modal")
+fun ModalBottomSheetStory(
+    modifier: Modifier = Modifier,
+) {
+    var openBottomSheet by rememberSaveable { mutableStateOf(true) }
+    val state = rememberBpkModalBottomSheetState()
+    Box(
+        modifier = modifier
+            .fillMaxSize()
+            .padding(top = BpkSpacing.Xxl),
+        contentAlignment = Alignment.Center,
+    ) {
+        BpkButton(text = stringResource(R.string.generic_show)) {
+            openBottomSheet = true
```

Comment:
Don't we suppose to use .show() method of the state here?

Reply:
show/hide behaves differently - both from an animation case (the bottom sheet is not coming in from the bottom) , but also it just hides it, but still retains the screen focus. even when using `state.isVisible` to decide to create the bottom sheet the behaviour is still wrong. i'm not sure when the show/hide functions _should_ actually be used tbh